docs/index.asciidoc

```diff
@@ -144,17 +174,97 @@ specify a custom set of response headers
   * Value type is <<boolean,boolean>>
   * Default value is `false`
 
-SSL Configurations
+Events are by default sent in plain text. You can
+enable encryption by setting `ssl` to true and configuring
+the `ssl_certificate` and `ssl_key` options.
+
+[id="plugins-{type}s-{plugin}-ssl_certificate"]
+===== `ssl_certificate` 
+
+  * Value type is <<path,path>>
+  * There is no default value for this setting.
+
+SSL certificate to use.
```

Comment:
Can we beef this up a little bit ?

I _think_ this is really a path to a PKCS#8 keystore ?